### DISCUSSION TITLE
Cannot create new projects due to free tier limit but have zero projects

### DISCUSSION THREAD

**Discussion by dsaltares** [supabase/supabase#9799] [Questions] [answered]
I am on the free tier and had created projects in the past. I have now deleted those projects and tried to create a new one. The UI doesn't let me and claims I have reached the limit (2). However, I have zero projects.

Has anyone hit this issue before?

<img width="759" alt="Screenshot 2022-10-24 at 12 57 25" src="https://user-images.githubusercontent.com/570314/197535200-04b67489-4b90-4936-b76c-b03e71904f84.png">
<img width="1919" alt="Screenshot 2022-10-24 at 12 57 17" src="https://user-images.githubusercontent.com/570314/197535206-5d888810-4a9b-4dd2-a459-7002ae594c52.png">

**Comment by GaryAustin1** [comment]
You should contact support while waiting to see if someone has hit this before and solved it on their own.

**Reply by dsaltares** [reply]
Thanks @GaryAustin1, just did it.

**Comment by dsaltares** [comment, marked as answer]
I contacted Supabase support and got this response.

> Thank you for contacting Supabase support.
It appears you have 2 projects stuck in a "GOING_DOWN" state. We're taking a look at the issue, I'll update you as soon as possible.

Looks like they fixed it and I can now create new projects. This was most likely a one-off, but if people experience this in their own accounts, they can now know they should reach out to support.

**Comment by mohakdev** [comment]
I am suffering through the same exact thing. Why exactly does this happen? How much time should I expect before I can create a new project cause it's urgent.

**Reply by GaryAustin1** [reply]
What is your exact situation?  Do you have projects locked up in pausing process like the OP?
Do you have team members in your ORG that have a total of more than 2 free projects?
